public/locales/en/home.json

```diff
@@ -1,5 +1,11 @@
 {
     "constructionNotice": "NOTE: This Website Is Under Construction. Check Back in April 2021.",
     "noActiveSensor": "Click a sensor on the map to see its AQI value here!",
-    "dataVizBox": "Data Visualization Box"
+    "dataVizBox": "Data Visualization Box",
+    "expandMap": "Expand to see the sensor map",
+    "expandAqiGauge": "Expand to see information about the selected sensor",
+    "expandAqiGraph": "Expand to see a graph of the last 24 hours",
+    "hideMap": "Compress to hide the sensor map",
```

Comment:
I think "Compress" isn't quite the right word, maybe "Click the arrow"? Or maybe we don't even need the label when it's expanded

Reply:
If we do want a label, I think collapse would be most appropriate.

Reply:
Changed to "collapse", will ask liaisons in the meeting if they want the label or not